source/tutorial/autocomplete-component.md

```diff
@@ -71,7 +71,7 @@ the new value of the property is present in both the rendered web page and in th
 
 The `key-up` property will be bound to the `handleFilterEntry` action.
 
-Here is what the component's JavaScript looks like:
+The `handleFilterEntry` action will assign `results` with the list of rentals it returns. The `{{yield}}` helper will then use `results` to fill in the value of our block parameter, `|filteredResults|`. We still need to define how `handleFilterEntry` gets those results though, so let's do that now:
```

Comment:
@BradenLawrence Nice work! I'm not sure I understand what you wrote here though. I'd be happy if we can work to get this PR merged in as early as possible since the corresponding change on Super Rentals is already live.

Reply:
Hi @yawboakye, my goal here is to explain how block form components and the {{yield}} tag interact. 

The original text does a good job explaining how code we put inside the {{myComponent}} ... {{/myComponent}} block goes to the {{yield}} tag, but it doesn't explain how code we put in {{yield myVariable}} is sent back to the parent. I was confused by this when I first went through the guides because I couldn't figure out where the data in "as |rentals|" was coming from.

I tried to address this by making the variable names match up: "{{yield results}}" feeds into "as |filteredResults|", and walking through it step by step on line 74. Is there a better way I could word this? Let me know if you have any questions!

Reply:
Maybe something like this?
```
The `handleFilterEntry` action will apply the search term filter to the list of rentals, and set a component attribute called `results`. The `results` are passed to the `{{yield}}` helper in the template. In the yielded block component, those same `results` are referred to as `|filteredResults|`. Let's apply the filter to our rentals:
```